docs/source/en/training/distributed_inference.md

```diff
@@ -52,6 +52,62 @@ To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](h
 
 </Tip>
 
+### Using `device_map`
+
+<Tip warning={true}>
+
+🧪 This feature is experimental and its APIs might change in future. 
+
+</Tip>
```

Comment:
Let's start using this format more since it takes up less space and is cleaner :)

```suggestion
> [!WARNING]
> This feature is experimental and its APIs might change in the future. 
```